gitops/openshift/apps/nginx/template/nginx_jag_scss.env

```diff
@@ -0,0 +1,7 @@
+APP_NAME=jag-scss
+COMPONENT_NAME=nginx-jag-scss
+OC_NAMESPACE=17b24e
```

Comment:
I've created a secret for this: OPENSHIFT_SA_NAMESPACE - should we use this instead?